docker-run.sh

```diff
@@ -6,13 +6,13 @@
 #
 
 source benchmark-scripts/get-gpu-info.sh
+# Default WORKLOAD_SCRIPT to dlstreamer workload if not provided
```

Comment:
what i meant is the comment of the relationship of this env value with --workload... as it is not very obvious this env will be overridden by that option below..kind of code maintainability.